### PR TITLE
Data 1830

### PR DESCRIPTION
**Pull request by ML-engineer**
Added processingTime attribute
Upgraded org.apache.httpcomponents dependencies to address vulnerabilities.

### CODE REVIEW

**Review by draimosson-mathem — DISMISSED** [review]
Looks good!

File: generic/cloudbuild.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# gcloud builds submit --config=./generic/cloudbuild.yaml . --substitutions=_VERSION=0.9.0
+# gcloud builds submit --config=./generic/cloudbuild.yaml . --substitutions=_VERSION=0.10.0
```
Comment (draimosson-mathem) [inline generic/cloudbuild.yaml:1]: You specify version 0.10.0 here... but further down and in the pom file it still says version 0.9.0

**Review by draimosson-mathem — CHANGES_REQUESTED** [review]
Apart from version numbers everything looks good!

File: generic/pom.xml [hunk 2]
```diff
@@ -219,7 +219,7 @@
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
-            <version>4.5.5</version>
+            <version>[4.5.13,)</version>
```
Comment (jesperdejby) [inline generic/pom.xml:222]: Thoughts for learning purposes, this means "versions after 4.5.13 (4.5.13 included)"? The soft bracket however indicates "exclusive"? So will it include or exclude the later version in that case? I.e. what happens if you write `[4.5.13,]`?

**Review by jesperdejby — COMMENTED** [review]
LGTM apart from Davids comment 👍

File: generic/cloudbuild.yaml (hunk 1 above)
Comment (ML-engineer) [reply generic/cloudbuild.yaml:1]: good catch

File: generic/pom.xml (hunk 2 above)
Comment (ML-engineer) [reply generic/pom.xml:222]: here's a good reference https://docs.oracle.com/middleware/1212/core/MAVEN/maven_version.htm#MAVEN402

**Review by jesperdejby — APPROVED** [review]
LGTM 👍

**Pull request merged**